buildscripts/tests/test_update_test_lifecycle.py

```diff
@@ -1009,6 +1009,15 @@ def test_clean_up_description_empty(self):
         expected = "_None_"
         self.assertEqual(expected, desc)
 
+    def test_truncate_description(self):
+        desc = "a" * (JiraIssueCreator._MAX_DESCRIPTION_SIZE - 1)
+        self.assertTrue(desc == JiraIssueCreator._truncate_description(desc))
```

Comment:
`JiraIssueCreator` is not defined here, you'll need to reference the class using `update_test_lifecycle.JiraIssueCreator`.

Same on lines 1017 and 1020.